File: Backend/Microservice-orders/Backend.DataBase/Models/OrderModel.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace Backend.Domain.Models
{
    public class OrderModel
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string IdOrder { get; set; }

        public string IdUser { get; set; }  
        public string NameApplicant { get; set; }
        public string EmailApplicant { get; set; }
        public List<string> Products { get; set; }
        public DateTime CreatedDate { get; set; }
        public bool Enabled { get; set; }
    }
}

File: Frontend/company-ABC/src/app/features/orders/models/order.model.ts
export interface Order {
    idOrder: string;
    idUser: string;
    nameApplicant: string;
    emailApplicant: string;
    products: string[];
}
export interface CreateOrder {
    idUser: string;
    idOrder: string;
    nameApplicant: string;
    emailApplicant: string;
    products: string[];

}
export interface DeleteOrder {
    idOrder: string;

}